docs/configuration.md

```diff
@@ -0,0 +1,151 @@
+| `extends`   | `string`\|`string[]`   | Either a string or an array of strings. Each string allows you to specify an internally curated list of rules (we suggest `recommended` here).                                                                                                                                                                                                                                                                                                                  |   |   |
+| `pending`   | `string[]`             | An array of module id's that are still pending. The goal of this array is to allow incorporating template linting into an existing project without changing every single template file. You can add all existing templates to this `pending` listing and slowly work through them while at the same time ensuring that new templates added to the project pass all defined rules. You can generate this list with the: `ember-template-lint * --print-pending`. |   |   |
+| `ignore`    | `string[]`\|`glob[]`   | An array of module id's that are to be completely ignored. See [ignore documentation](docs/ignore.md) for more details.                                                                                                                                                                                                                                                                                                                                         |   |   |
+| `plugins`   | `string[]`\|`object[]` | An array of plugin objects or strings that resolve to files that export plugin objects. See [plugin documentation](docs/plugins.md) for more details.                                                                                                                                                                                                                                                                                                           |   |   |
+| `overrides` | `Array`                | An array of overrides that would allow overriding of specific rules for user specified files/folders. See [overrides documentation](docs/overrides.md) for more details.                                                                                                                                                                                                                                                                                        |   |   |
+## Severity Levels
```

Comment:
The linting failure is due to missing a newline above this heading

```suggestion

## Severity Levels
```

Reply:
It's fixed.